src/xcape/core.py

```diff
@@ -132,81 +138,83 @@ def calc_cape(*args, **kwargs):
-        can be influenced by moisture depth. Mixed-layer parcels are generally a good assumption
-        for profiles at peak heating when the boundary layer is deeply mixed to approximately the
-        boundary layer depth. Most-unstable is defined by the layer below 500hPa with the highest 
-        equivalent potential temperature. 
-    ml_depth : float, optional
+    source : str, optional, default is 'surface'
+        Select parcel based on desired assumptions under parcel theory
+            - 'surface' = Surface-based parcels are subject to substantial errors depending on surface heating and source data, and can be influenced by moisture depth.
+            - 'mixed-layer' = Mixed-layer parcels are generally a good assumption for profiles at peak heating when the boundary layer is deeply mixed to approximately the boundary layer depth.
+            - 'most-unstable' = Most-unstable is defined by the layer below 500hPa with the highest equivalent potential temperature.
+    ml_depth : float, optional (default is 500)
         Depth (m) of mixed layer. Only applies when the source='mixed-layer'
-    adiabat : {'pseudo-liquid', 'reversible-liquid','pseudo-ice', 'reversible-ice'}
-    pinc : float, optional
+    adiabat : str, optional, default is 'pseudo-liquid'
+        options include: 'pseudo-liquid', 'reversible-liquid','pseudo-ice', 'reversible-ice'  
+    pinc : float, optional, default is 500
         Pressure increment for integration (Pa) - Recommended usage (between 1000 and 100) is 
         based on desired speed, with accuracy of the calculation increasing with smaller 
         integration increments. 
-    method : {'fortran', 'numba'}
-        Option to select numerical approach using wrapped Fortran 90 or a Numba python variant.
-    vertical_lev : {'sigma', 'pressure'}
-        Option to select vertical grid, between model coordinates and pressure levels.
+    method : str, optional, default is 'fortran')
+        Option to select numerical approach using wrapped Fortran 90 or a Numba python variant (to be implemented)
+    vertical_lev : str, optional, default is 'sigma'
+        Option to select vertical grid, between model coordinates, 'sigma', and pressure levels, 'pressure'.
 
     Returns
     -------
-    cape : 'array-like'
+    cape : array-like
         Convective available potential energy (J/Kg)
-    cin : 'array-like'
+    cin : array-like
         Convective inhibition (J/Kg)
-    MUlev : 'array-like'
+    MUlev : array-like
         Most Unstable level location index (only returned for source: {'most-unstable'})
-    zMUlev : 'array-like'
+    zMUlev : array-like
         height of MUlev (m) (only returned for source: {'most-unstable'}
-     """
+        
+    Notes
+    -----
+    CAPE is calculated on a user specified set of parcel options based on the integration:
+    
+    
+    .. math:: CAPE = g \\int_{LFC}^{EL} (\Theta_{v,parcel} - \Theta_{v,env}/(\Theta_{v,env}) d\\text{dz}
```

Comment:
Should be \text{dz} not d\text{dz} here and the subsequent line.
Can we change the theta symbol - this one is confusing with Geopotential?

Reply:
done!